zaza/openstack/charm_tests/glance/tests.py

```diff
@@ -67,6 +68,38 @@ def test_411_set_disk_format(self):
             {'image_format': {'disk_formats': ['qcow2']}},
             ['glance-api'])
 
+    def test_412_image_conversion(self):
```

Comment:
Test cases should make an effort to restore the system under test to how it was before the test was run, so that the order in which all the tests are run doesn't really matter. So this test should revert the config change at the end I would say, if possible.

This is something we do a lot in other tests and it looks like:

```
with self.config_change(...):
    ...
```

What do you think?

Reply:
Great advice! thanks :)